examples/Interceptor/Client/ClientLoggerInterceptor.cs

```diff
@@ -132,5 +165,16 @@ private async Task<TResponse> HandleResponse<TResponse>(Task<TResponse> t)
             headers.Add("caller-machine", Environment.MachineName);
             headers.Add("caller-os", Environment.OSVersion.ToString());
         }
+
+        private static void LogError(Exception ex)
+        {
+            // Log error to the console.
+            // Note: Configuring .NET Core logging is the recommended way to log errors
+            // https://docs.microsoft.com/aspnet/core/grpc/diagnostics#grpc-client-logging
```

Comment:
Why not configure the sample to use the logging primitives then? Looks like it should be really easy.